src/types/contact-request.ts

```diff
@@ -1,10 +1,10 @@
 /*
```

Comment:
Renamed the path because `types` as a name can also encompass `interface` declarations, in my head - but not the other way around.

Reply:
Makes sense to me!